### PR TITLE
Sachinsom93/gosw20

### PR DESCRIPTION
**Pull request by sachinsom93**
#### Issue Number
ISSUE #52 

#### Describe the changes you've made
Loaded events from firestore to upcoming events page.
#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

#### Test plan (OPTIONAL)
A good test plan should give instructions that someone else can easily follow.
How someone can test your code?

#### Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](https://github.com/GameofSource-GFG/Web-Development/blob/main/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Comment by sachinsom93** [comment]
Use this code  in App.js to check:
```
import React from "react";
import logo from "./logo.svg";
import "./App.css";

import AboutUs from "./components/AboutUs/AboutUs";
import Commiittee from "./components/CommitteeMembers/committee";
import Homepage from "./components/Homepage/Homepage";
import Timeline from "./components/Timeline/timeline";
import ContactUs from "./components/Contact-Us/ContactUs";
import SocialMediaHandles from './components/SocialMediaHandles'
import MisVis from "./components/Mission&Vision/MisVis";
import ShowAllEvents from './components/EventForm/ShowEvents/ShowAllEvents'

function App() {
  return (
    <>
      <ShowAllEvents />
    </>

  );
}
export default App;

```

File: client/src/components/EventForm/ShowEvents/ShowOneEvent.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    fontWeight: "lighter",
+    left: "0",
+    testAlign: "center",
+    marginBottom: "1.5rem"
+  }
+});
+
+
+function ShowOneEvent(props) {
+    const classes = useStyles()
```
Comment (07souravkunda) [inline client/src/components/EventForm/ShowEvents/ShowOneEvent.jsx:52]: Use object destructuring on props to assign variables.

Comment (sachinsom93) [reply client/src/components/EventForm/ShowEvents/ShowOneEvent.jsx:52]: OK

Comment (sachinsom93) [reply client/src/components/EventForm/ShowEvents/ShowOneEvent.jsx:52]: > Use object destructuring on props to assign variables.

@07souravkunda done, Review it again plz,

**Review by 07souravkunda — APPROVED** [review]
good work @sachinsom93 . Everything seems fine, I am accepting this PR.

**Pull request merged**